allwinner/h6.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// This file contains pin mapping information that is specific to the Allwinner
+// H5 model.
+
+package allwinner
+
+import (
+	_ "embed"
+	"encoding/json"
+	"periph.io/x/conn/v3/pin"
```

Comment:
I recommend to setup goimports in your IDE so you don't have to type them manually ever again. This will sort the entries correctly automatically.

Reply:
I do not understand what is wrong here.  The imports are ordered alphabetically, it is the default of my "goland" ide.  `gofmt` seems to do the same.  Here I am forced to import `embed` as I use it to load the pin spec,

Reply:
It's not a big deal. It's just the first time I see someone using goland's way of sorting imports.